packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta6-12032" />
   <package id="dnx-mono" version="1.0.0-beta5-12077" />
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00053" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00085" />
```

Comment:
DNU --- it will need revisiting now that packages contain nested runtime.json for deployment packages.